tests/runtime/ray/conftest.py

```diff
@@ -0,0 +1,29 @@
+################################################################################
+# © Copyright 2022 Zapata Computing Inc.
+################################################################################
+import os
+
+import pytest
+
+from orquestra.sdk._base._testing import _connections
+
+
+@pytest.fixture(scope="module")
+def change_test_dir(tmp_path_factory, request):
+    project_dir = tmp_path_factory.mktemp("project")
+    os.chdir(project_dir)
+    yield project_dir
+    os.chdir(request.config.invocation_dir)
+
+
+@pytest.fixture(scope="module")
+def shared_ray_conn():
+    with _connections.make_ray_conn() as ray_params:
+        yield ray_params
+
+
+@pytest.fixture(scope="module")
+def change_db_location(change_test_dir):
+    os.environ["ORQ_DB_LOCATION"] = os.path.join(change_test_dir, "db.db")
+    yield
+    del os.environ["ORQ_DB_LOCATION"]
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion (non-blocking):** ‏Wdyt of:
```suggestion
@pytest.fixture(scope="module")
def change_db_location(change_test_dir, monkeypatch):
    monkeypatch.setenv("ORQ_DB_LOCATION", os.path.join(change_test_dir, "db.db")
```

Reply:
@jamesclark-Zapata does it actually work? I thought monkeypatch is function-scoped fixture and wont work when used in module-scoped fixture

Reply:
I think you're right!